src/Microsoft.Data.Analysis/DoubleConverter.cs

```diff
@@ -30,7 +30,7 @@ public static IDoubleConverter<T> GetDoubleConverter<T>()
             }
             else if (typeof(T) == typeof(decimal))
             {
-                throw new NotImplementedException();
+                return (IDoubleConverter<T>)new DecimalDoubleConverter();
```

Comment:
Is there a test for this change?

Reply:
Added one now. We didn't have 1 before because there was no way to reach this code.